### ISSUE TITLE
onPress isn't working

### ISSUE THREAD

**Issue by dayvsonsales** [GeekyAnts/react-native-easy-grid#46]
Hello, 
I'm testing your library and trying to make a simple app with 4 clickable items. But when I put the onPress tag inside row or col component doesn't show anything.
My code is:

```javascript
import React from 'react';
import { StyleSheet, Text, View, Alert, TouchableOpacity } from 'react-native';
import { Col, Row, Grid } from "react-native-easy-grid";

export default class App extends React.Component {
  render() {
    return (
      <Grid>
        <Col>
          <Col  onPress={ this.alerta} accessibilityLabel="White" accessibility={true} style={{backgroundColor: '#fff'}}></Col>
          <Row  onPress={ this.alerta} accessibilityLabel="Red" accessibility={true} style={{backgroundColor: 'red'}}></Row>
        </Col>
        <Col>
          <Row  onPress={ this.alerta} accessibilityLabel="Green" accessibility={true} style={{backgroundColor: '#2f5'}}></Row>
          <Row  onPress={ this.alerta} accessibilityLabel="Blue" accessibility={true} style={{backgroundColor: '#6ff'}}></Row>
        </Col>
      </Grid>
    );
  }
  alerta(){
       Alert.alert('testing');
  }
}
```

**Comment by singleseeker** [comment]
Put onPress in your child element. It will be work. It`s not common put the event on layout element.

**Comment by SupriyaKalghatgi** [comment]
Fixed with 0.1.17
